llpc/translator/lib/SPIRV/libSPIRV/SPIRVModule.cpp

```diff
@@ -484,6 +486,27 @@ void SPIRVModuleImpl::optimizeDecorates() {
   }
 }
 
+void SPIRVModuleImpl::postProcessExecutionModeId() {
+  for (auto ExecModeId : ExecModeIdVec) {
+    SPIRVExecutionModeId *E = static_cast<SPIRVExecutionModeId *>(ExecModeId);
+    auto M = E->getExecutionMode();
+    switch (M) {
+    case ExecutionModeLocalSizeId: {
+      auto TId = E->getTargetId();
+      auto Ops = E->getOperands();
+      auto workGroupSizeX = static_cast<SPIRVConstant *>(getEntry(Ops[0]));
```

Comment:
In this file, we still stick to the style of existing codes. So change workGroupSize to WorkGroupSize